pkg/render/yaml.go

```diff
@@ -27,9 +27,32 @@ func (r YAML) File(_ context.Context, _ *malcontent.FileReport) error {
 }
 
 func (r YAML) Full(_ context.Context, rep *malcontent.Report) error {
-	// Drop the applied filters
-	rep.Filter = ""
-	yaml, err := yaml.Marshal(rep)
+	// Make the sync.Map YAML-friendly
+	type yamlReport struct {
```

Comment:
This looks like a duplicate of `jsonReport` in `pkg/render/json.go` - use the same object?

Reply:
Updated in [`f2d2599` (#617)](https://github.com/chainguard-dev/malcontent/pull/617/commits/f2d2599b9505783c903b658b6b396c1c9b19b786).